util/python/python_config.sh

```diff
@@ -16,7 +16,11 @@
 
 set -e -o errexit
 
-EXPECTED_PATHS="util/python/python_include util/python/python_lib third_party/py/numpy/numpy_include"
+# Prefix expected paths with ./ locally and external/reponame/ for remote repos.
+# TODO: remove once runfiles paths are fixed.
```

Comment:
Can you add yourself to the TODO?  maybe link to a github issue for the runfile paths?